Source/WebCore/dom/Element.h

```diff
@@ -660,7 +660,8 @@ class Element : public ContainerNode {
     ExceptionOr<Ref<WebAnimation>> animate(JSC::JSGlobalObject&, JSC::Strong<JSC::JSObject>&&, std::optional<std::variant<double, KeyframeAnimationOptions>>&&);
     Vector<RefPtr<WebAnimation>> getAnimations(std::optional<GetAnimationsOptions>);
 
-    ElementIdentifier createElementIdentifier();
+    WEBCORE_EXPORT ElementIdentifier identifier() const;
```

Comment:
This function has the side effect of adding the element into an identifier map, a bit costly. Given that, maybe we want it to have a name that sounds more like it’s doing work rather than just returning some information. On the other hand, it’s good that this function has the same name as the one on the document, which has the same issue. There are so many more elements than documents that it does seem important to avoid making the mistake of calling this too much. I don’t think we have that mistake right now, although I’m unclear on how many of these we might accumulate due to `regionFromElement`.